src/server/pkg/worker/master.go

```diff
@@ -388,7 +390,7 @@ func (a *APIServer) runJob(ctx context.Context, jobInfo *pps.JobInfo, pool *pool
 		}
 
 		failed := false
-		limiter := limit.New(a.numWorkers)
+		limiter := limit.New(a.numWorkers * 10)
```

Comment:
Not sure if I'm missing something, but why is this 10 while the other number is 5?